bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/URLAudioStream.java

```diff
@@ -90,7 +90,10 @@ private InputStream createInputStream() throws AudioException {
                 os.write(req.getBytes());
                 is = shoutCastSocket.getInputStream();
             } else {
-                is = streamUrl.openStream();
+                // getInputStream() method is more error-proof than openStream(),
+                // because openStream() does openConnection().getInputStream(), 
+                // which opens a new connection and does not reuse the old one.
+                is = connection.getInputStream();
```

Comment:
Yes, I think we can call this a bug. `openStream()` method calls `openConnection().getInputStream()`, which opens an entirely new connection and does not reuse the old one. Also, `openStream()` initiates HTTP GET request. For example, this may cause a problem when we use `oneTimeStreams` in the `AudioServlet`, because `oneTimeStreams` can be requested a single time. 

This problem can be solved, if we use `getInputStream()` method on the currently opened connection.

I put it in this PR, because it is a small change, but if you want I can make a separate PR with this change.